roles/nova-data/tasks/v7k.yml

```diff
@@ -0,0 +1,42 @@
+---
+- block:
+  - name: install multipath-tools packages (ubuntu)
+    package: name={{ item }}
+    with_items:
+      -  multipath-tools
+    register: result
+    until: result|succeeded
+    retries: 5
+
+  - name: generate multipath.conf
+    template: src=etc/multipath.conf dest=/etc/multipath.conf
+              mode=0640
+              owner=root group=root
+
+  - name: restart multipath-tools
+    service:
+      name:  multipath-tools
+      state: restarted
+  when: ursula_os == 'ubuntu'
+
+
+- block:
+  - name: install device-mapper-multipath packages
+    package: name={{ item }}
+    with_items:
+      - device-mapper-multipath
+    when: ursula_os == 'rhel'
+    register: result
+    until: result|succeeded
+    retries: 5
+
+  - name: generate multipath.conf
+    template: src=etc/multipath.conf dest=/etc/multipath.conf 
+              mode=0640
+              owner=root group=root
+
+  - name: restart multipathd
```

Comment:
when we are installing service, we should always have start service task  (not restart) ...also it is important to ensure service is enabled to start on reboot.. see example @ https://github.com/blueboxgroup/ursula/blob/master/roles/nova-data/tasks/main.yml#L116